server/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -245,61 +228,47 @@ private static void initPhase2(Bootstrap bootstrap) throws IOException {
 
         final PluginsLoader pluginsLoader;
 
-        if (bootstrap.useEntitlements()) {
```

Comment:
We also have a `useEntitlements()` check in `initPhase3()`, can that be removed as well? In fact, can we remove `Bootstrap.useEntitlements()` entirely now?

Reply:
Actually, I'll remove that in a followup as well.